Petty/Extensions/UIColor+.swift

```diff
@@ -9,5 +9,9 @@
 import UIKit
 
 extension UIColor {
-    static let frenchBlue = UIColor(red: 0.0 / 255.0, green: 109.0 / 255.0, blue: 188.0 / 255.0, alpha: 1.0)
+    convenience init(r: CGFloat, g: CGFloat, b: CGFloat, a: CGFloat) {
```

Comment:
a: CGFloat?

Reply:
@property (readonly) CGFloat alpha;
CGFloat mà e

Reply:
đúng r em, alpha là CGFloat.
/* Return the alpha value of the color. */
@property (readonly) CGFloat alpha;